src/topojson/topojson.ml

```diff
@@ -380,6 +380,8 @@ module Make (J : Intf.Json) = struct
       objects : (string * Geometry.t) list;
       arcs : Geometry.Position.t array array;
       foreign_members : (string * json) list;
+      transform:  (string* Geometry.Position.t array )list;
```

Comment:
I think we can more specific here given the specification: 

> A transform must have a member with the name “scale” whose value is a two-element array of numbers.
> A transform must have a member with the name “translate” whose value is a two-element array of numbers.

So I think we can define a new type for the transform object. 

```
type transform = {
  scale : float * float;
  translate : float * float
}
```

Note we'll have to do a conversion by hand from a JSON array of numbers to a pair of OCaml floats. 

Also, it should be optional i.e. `transform : transform option`